contracts/hydro/src/testing_lsm_integration.rs

```diff
@@ -736,7 +740,30 @@ fn lock_tokens_multiple_validators_and_vote() {
     }
 
     // update the power ratio for validator 1 to become 0.5
-    set_validator_power_ratio(deps.as_mut().storage, 0, VALIDATOR_1, Decimal::percent(50));
+    let mock_tokens = Uint128::new(500);
```

Comment:
could this be a util function somewhere? I know it's not really used anywhere else right now, but it seems like the type of code block that should be hidden in a function :)

Reply:
Not sure what do you mean by that. `get_mock_validator()` is already a util function, but we want to keep it flexible, so the tokens and shares are provided by the caller. We could add a `shares_from_tokens()`, perhaps this is what you meant?

Reply:
the whole block up to the sudo call is what I meant

Reply:
Done in https://github.com/informalsystems/hydro/pull/220/commits/400c1abb6ebaa9c98891b9ac0a71b262c6aea30e